scripts/tutorial-uploader/lib/api.test.ts

```diff
@@ -151,17 +134,21 @@ describe("Tutorial uploader API", () => {
     ).translations;
     expect(retrievedTranslation).toMatchObject([
       {
-        title: simpleTutorial.title,
-        short_description: simpleTutorial.short_description,
+        title: modifiedTutorial.title,
+        short_description: modifiedTutorial.short_description,
         content: "Here's some basic content.\n",
         languages_code: "en-US",
       },
     ]);
   });
 
-  afterAll(async () => {
-    for (const slug of createdSlugs) {
-      await api.deleteTutorial(slug);
+  beforeAll(async () => {
+    if (await api.getId("tutorials", "slug", MOCK_TUTORIAL.slug)) {
```

Comment:
Nice rewrite! This line is now repeated several times and could be a helper function